software/introduction.qmd

```diff
@@ -155,9 +161,9 @@ Seedcase is developed and designed with four users and uses cases in mind. The f
 
 ### Main functionality
 
--   **One Seedcase "instance" per Data Resource**: Some users may have
-    multiple Data Resources, so the Seedcase Product will create
-    separate "instances" for each Data Resource.
+-   **One Data Resource per Seedcase "instance"**: For users with
+    multiple Data Resources, each Data Resource will be housed in a
+    separate "instance" of the infrastructure framework.
```

Comment:
For me, it makes more sense this way, but I may have misunderstood it!

Reply:
Yea, this will probably be updated in the next iteration, I think it is partly out of date, or at least not completely accurate.